presentations/2024-10-10_what-is-ai-tom/index.qmd

```diff
@@ -0,0 +1,434 @@
+> human capabilities**.
+:::
+
+::: {.fragment}
+> In this case, computer vision seeks to replicate both the way **humans see**, and
+> the way humans **make sense** of what they see.
+:::
+
+::: footer
+Source: [What is computer vision? Microsoft](https://azure.microsoft.com/en-gb/resources/
+cloud-computing-dictionary/what-is-computer-vision#object-classification)
+:::
+
+# What tasks can CV be used for? {.center .inverse}
+
+---
+
+## Classification
+
+::: {.fragment fragment-index=6}
+*Assign a single label to each image*
+:::
+
+:::: {.columns}
+
+::: {.column width="40%"}
+
+::: {.fragment fragment-index=1}
+![](https://raw.githubusercontent.com/EliSchwartz/imagenet-sample-images/refs/heads/master/n02102177_Welsh_springer_spaniel.JPEG){height=300px}
+:::
+
+::: {.fragment fragment-index=3}
+Dog
+:::
+::: {.fragment fragment-index=4}
+Welsh Spaniel
+:::
+::: {.fragment fragment-index=5}
+Animial in water
+:::
+
+:::
+
+::: {.column width="20%"}
+:::
+
+::: {.column width="40%"}
+
+::: {.fragment fragment-index=2}
+![](https://raw.githubusercontent.com/EliSchwartz/imagenet-sample-images/refs/heads/master/n02102480_Sussex_spaniel.JPEG){height=300px}
+:::
+
+::: {.fragment fragment-index=3}
+Dog
+:::
+::: {.fragment fragment-index=4}
+Sussex Spaniel
+:::
+::: {.fragment fragment-index=5}
+Animial on land
+:::
+
+:::
+
+::::
+
+::: footer
+Images from [imagenet](https://www.image-net.org/)
+:::
+
+
+## Multi-Classification
+
+
+::: {.fragment fragment-index=2}
+*Assign one or more labels to each image*
+:::
+
+:::: {.columns}
+
+::: {.column width="40%"}
+
+![](https://raw.githubusercontent.com/EliSchwartz/imagenet-sample-images/refs/heads/master/n02102177_Welsh_springer_spaniel.JPEG){height=300px}
+
+::: {.fragment fragment-index=1}
+Dog, Welsh Spaniel, Animial in water
+:::
+
+:::
+
+::: {.column width="20%"}
+:::
+
+::: {.column width="40%"}
+
+![](https://raw.githubusercontent.com/EliSchwartz/imagenet-sample-images/refs/heads/master/n02102480_Sussex_spaniel.JPEG){height=300px}
+
+::: {.fragment fragment-index=1}
+Dog, Sussex Spaniel, Animial on land
+:::
+
+:::
+
+::::
+
+::: footer
+Images from [imagenet](https://www.image-net.org/)
+:::
+
+## Object Detection
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Detected-with-YOLO--Schreibtisch-mit-Objekten.jpg/1119px-Detected-with-YOLO--Schreibtisch-mit-Objekten.jpg)
+
+::: footer
+Image sourced from [Wikimedia](https://commons.wikimedia.org/wiki/File:Detected-with-YOLO--Schreibtisch-mit-Objekten.jpg)
+:::
+
+## Event Detection
+
+![](https://www.autonomousvehicleinternational.com/wp-content/uploads/2022/05/AdobeStock_215808766-1024x596.jpeg)
+
+::: footer
+[Is camera-only the future of self-driving cars?](https://www.autonomousvehicleinternational.com/features/is-camera-only-the-future-of-self-driving-cars.html)
+:::
+
+## How does CV work? { .inverse}
+
+### for classification tasks
+
+::: {.incremental}
+- get a very large corpus or labelled images
```

Comment:
```suggestion
- get a very large corpus of labelled images
```
Is corpus too technical? maybe dataset?

Reply:
I've gone with corpus because while dataset is probably correct, I think a lot of people will assume dataset = tabular data